include/cutlass/numeric_conversion.h

```diff
@@ -2690,6 +2691,77 @@ struct FastNumericArrayConverter<cutlass::half_t, uint8_t, 4, Round> {
   }
 };
 
+/// Partial specialization for Array<cutlass::half_t, 8> <= Array<int4b_t, 8>
+template <FloatRoundStyle Round>
+struct FastNumericArrayConverter<cutlass::half_t, cutlass::int4b_t, 8, Round> {
```

Comment:
No problem.  But this PR needs work anyway, and it certainly won't be landed before these changes of yours, so I'm keeping it for now just so that the rest of it works; as soon as your changes merged, I'll rebase and remove it.

Reply:
Ahh ok, it sounds good. :) How much work do you think is needed? Is it ready for review?

Reply:
The list of what remains to be done is in the first comment above.  I'd appreciate a feedback of what is already there, and eventually some suggestions about item 1 from this list.  Items 2 and 3 should be quick to complete afterwards.

Reply:
Makes sense. I am not sure of how to handle it apart from a fresh threadblock level mma where we skip loading some of the int4 data from smem